mimic/rest/auth_api.py

```diff
@@ -88,35 +90,33 @@ def lookup(entry):
                                   exception.session.username,
                                   exception.session.user_id))
 
-        if content['auth'].get('passwordCredentials'):
-            username = content['auth']['passwordCredentials']['username']
-            password = content['auth']['passwordCredentials']['password']
-            return format_response(
-                lambda: self.core.sessions.session_for_username_password(
-                    username, password, tenant_id),
-                username_generator)
+        cred_mappings = {
```

Comment:
Apparently multiple authentications are cool.  But `tenantId` + `tenantName` is not.